Dockerfile.train

```diff
@@ -1,39 +1,67 @@
-# Please refer to the TRAINING documentation, "Basic Dockerfile for training"
+# This is a Dockerfile useful for training models with Coqui STT.
+# You can train "acoustic" models with audio + Tensorflow, and
+# you can train "language" models with text + KenLM.
+
+FROM ubuntu:20.04 AS kenlm-build
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential cmake libboost-system-dev \
+    libboost-thread-dev libboost-program-options-dev \
+    libboost-test-dev libeigen3-dev zlib1g-dev \
+    libbz2-dev liblzma-dev
```

Comment:
```suggestion
RUN apt-get update && \
    apt-get install -y --no-install-recommends \
    build-essential cmake libboost-system-dev \
    libboost-thread-dev libboost-program-options-dev \
    libboost-test-dev libeigen3-dev zlib1g-dev \
    libbz2-dev liblzma-dev && \
    rm -rf /var/lib/apt/lists/*
```

Reply:
this is a multistage Dockerfile, no need to remove intermediate assets because they are lost with the previous stages

Reply:
They're still saved as layers.